verifier/trustpolicy/trustpolicy.go

```diff
@@ -267,23 +267,19 @@ func (trustPolicyDoc *Document) GetApplicableTrustPolicy(artifactReference strin
 
 // LoadDocument loads a trust policy document from a local file system
 func LoadDocument() (*Document, error) {
-	jsonFile, err := dir.ConfigFS().Open(dir.PathTrustPolicy)
+	policyDocument := &Document{}
+
+	err := config.Load(dir.PathTrustPolicy, policyDocument)
 	if err != nil {
 		switch {
 		case errors.Is(err, os.ErrNotExist):
 			return nil, fmt.Errorf("trust policy is not present. To create a trust policy, see: %s", trustPolicyLink)
 		case errors.Is(err, os.ErrPermission):
 			return nil, fmt.Errorf("unable to read trust policy due to file permissions, please verify the permissions of %s", filepath.Join(dir.UserConfigDir, dir.PathTrustPolicy))
 		default:
-			return nil, err
+			return nil, fmt.Errorf("malformed trust policy. To create a trust policy, see: %s", trustPolicyLink)
```

Comment:
This error message is not accurate. If the path is a symlink or other errors, the detail of the error is conveyed by "malformed trust policy".